builtin/mainmenu/init.lua

```diff
@@ -78,7 +78,31 @@ local function init_globals()
 		end,
 		-- Filter function
 		function(element, gameid)
-			return element.gameid == gameid
+			if element.gameid == gameid then
+				return true
+			end
+			local gameconfig = Settings(pkgmgr.find_by_gameid(gameid).path .. "/game.conf")
+			local aliases = (gameconfig:get("gameid_alias") or ""):split()
+			for _, alias in pairs(aliases) do
+				alias = alias:trim()
+				if element.gameid == alias then
+					return true
+				else
```

Comment:
You don't need to indent this block, you already have an early return right above it. Get rid of the redundant `else`, otherwise the indentation gets unnecessarily deep.